game.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <string>
+using namespace std;
```

Comment:
I've seen very well made arguments for why to not `using namespace std` globally. (e.g. [TheCherno](https://www.youtube.com/watch?v=4NYC-VU-svE), [Medium](https://medium.com/@manishjaiswal/what-does-using-namespace-std-mean-in-c-fb4bac916e3e), [geeksforgeeks](https://www.geeksforgeeks.org/using-namespace-std-considered-bad-practice/))

I don't think we should teach this or sponsor it, IMO.